core/chains/evm/logpoller/log_poller.go

```diff
@@ -659,11 +659,9 @@ func (lp *logPoller) backfill(ctx context.Context, start, end int64) error {
 		gethLogs, err := lp.ec.FilterLogs(ctx, lp.Filter(big.NewInt(from), big.NewInt(to), nil))
 		if err != nil {
 			var rpcErr client.JsonError
-			if errors.As(err, &rpcErr) {
-				if rpcErr.Code != jsonRpcLimitExceeded {
-					lp.lggr.Errorw("Unable to query for logs", "err", err, "from", from, "to", to)
-					return err
-				}
+			if !errors.As(err, &rpcErr) || rpcErr.Code != jsonRpcLimitExceeded {
```

Comment:
quoting Jordans comment from log poller fix err handling PR
Should we express this in a cleaner way
`if !(errors.As(err, &rpcErr) && rpcErr.Code == jsonRpcLimitExceeded)`

Reply:
@reductionista I think this is your change.

Reply:
We should just drop this commit from this branch, since it's going in a separate PR and is unrelated to ChainReader

Reply:
Cool, if you check it in, I can merge dev again to Next and this branch, and then it'll disappear :).

Reply:
Or if you can tell me which files(s) to revert I can do that.